ext/curl/curl_file.stub.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class CURLFile {
+    public function __construct(
+        string $filename,
+        string $mimetype = UNKNOWN,
+        string $postname = UNKNOWN
+    ) {}
+
+    public function getFilename(): string {}
```

Comment:
For BC reasons, class methods need to use `@return` for now, even if it's just a single return type.

Reply:
for all of them or only this one?

Reply:
For all methods (it's fine for functions, but not methods).